cmd/antrea-agent/agent.go

```diff
@@ -127,7 +128,16 @@ func run(o *Options) error {
 
 	go agentMonitor.Run(stopCh)
 
+	serverOpt, err := antctl.NewServerOptions(antctl.Definition, agentMonitor, nil, antctl.DefaultAgentLocalPort)
```

Comment:
Joe suggested to use USD for local endpoints for security consideration. But we could address that in a new PR later.

Reply:
I also prefer to use a USD. Since the agent is a daemon set, the port would be exposed directly on host.

Reply:
@jianjuns, I have changed to use UDS, PTAL.